website/docs/ingestion_flink.md

```diff
@@ -1,179 +1,361 @@
+data‑warehouse ETL pipeline (incremental computing). Hudi MOR tables store messages as rows, which supports the retention of all change logs (integration at the format level).
+All changelog records can be consumed with the Flink streaming reader.
 
-|  Option Name  | Required | Default | Remarks |
-|  -----------  | -------  | ------- | ------- |
-| `changelog.enabled` | `false` | `false` | It is turned off by default, to have the `upsert` semantics, only the merged messages are ensured to be kept, intermediate changes may be merged. Setting to true to support consumption of all changes |
+### Options
+
+| Option Name         | Required | Default | Remarks                                                                                                                                                                                                 |
+|---------------------|----------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `changelog.enabled` | `false`  | `false` | It is turned off by default, to have the `upsert` semantics, only the merged messages are ensured to be kept, intermediate changes may be merged. Setting to true to support consumption of all changes |
 
 :::note
-Batch (Snapshot) read still merge all the intermediate changes, regardless of whether the format has stored the intermediate changelog messages.
+Batch (snapshot) reads still merge all the intermediate changes, regardless of whether the format has stored the intermediate changelog messages.
 :::
 
 :::note
-After setting `changelog.enable` as `true`, the retention of changelog records are only best effort: the asynchronous compaction task will merge the changelog records into one record, so if the
-stream source does not consume timely, only the merged record for each key can be read after compaction. The solution is to reserve some buffer time for the reader by adjusting the compaction strategy, such as
-the compaction options: [`compaction.delta_commits`](#compaction) and [`compaction.delta_seconds`](#compaction).
+After setting `changelog.enable` to `true`, the retention of changelog records is best‑effort only: the asynchronous compaction task will merge the changelog records into one record, so if the
+stream source does not consume in a timely manner, only the merged record for each key can be read after compaction. The solution is to reserve buffer time for the reader by adjusting the compaction strategy, such as
+the compaction options `compaction.delta_commits` and `compaction.delta_seconds`.
 :::
 
+## Append Mode
+
+For `INSERT` mode write operations, the current workflow is:
+
+- For Merge‑on‑Read tables, auto‑file sizing is enabled by default
+- For Copy‑on‑Write tables, new parquet files are written directly; auto-file sizing is not applied
```

Comment:
The MOR behavior is now changed to be the same with COW: new parquet files are written directly; auto-file sizing is not applied